.spelling

```diff
@@ -0,0 +1,58 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+DSNPArchives
+DSNPMessages
+DSNP
+URIs
+TBD
+blockchain
+Ethereum
+secp256k1
+keccak-256
+DSNPBatch
+dsnpHash
+dsnpUri
+DSNPAnnounce
+ERC165
+testnet
+mainnet
+RLP
+eth2
+CBOR
+GraphChange
+KeyList
+keyList
+PrivateGraphKeylist
+EncryptionKeyList
+messageID
+dsnpData
+inReplyTo
+PrivateGraph
+PrivateGraphChange
+EncryptedInbox
+PrivateBroadcast
+spec
+Dokz
```

Comment:
I feel like Dokz shouldn't be anywhere that counts as spell checking, but perhaps not?

Reply:
it's probably irrelevant now since it's Docz

Reply:
I misspoke.  Docz is in the README, which should still be spellchecked.